src/java.base/share/classes/sun/security/ssl/ServerHello.java

```diff
@@ -794,6 +794,15 @@ public byte[] produce(ConnectionContext context,
             hhrm.write(shc.handshakeOutput);
             shc.handshakeOutput.flush();
 
+            // In the Middlebox Compatibility Mode the server sends a
```

Comment:
maybe this reads better:
// In TLS1.3 middlebox compatibility mode the server sends a